lib/engine/game/g_1849.rb

```diff
@@ -42,9 +42,14 @@ class G1849 < Base
       SELL_MOVEMENT = :down_per_10
       POOL_SHARE_DROP = :one
 
-      MARKET_TEXT = Base::MARKET_TEXT.merge(phase_limited: 'Can only enter during phase 16').freeze
+      MARKET_TEXT = Base::MARKET_TEXT.merge(phase_limited: 'Can only enter during phase 16',
+                                            par: 'Yellow phase par',
+                                            par_1: 'Green phase par',
+                                            par_2: 'Brown phase par').freeze
       STOCKMARKET_COLORS = {
         par: :yellow,
+        par_1: :green,
+        par_2: :brown,
```

Comment:
Need to check this'll actually show up in par values. See 1867 buy_sell_par_shares.rb

Reply:
I followed the 1828 model, which doesn't have a `get_all_par_prices` method and seems to work fine. When I tested it, I was not able to use the green par box at the start of the game but was able to once we entered the green phase. I was mostly just copying from 1828 code so I'm not at all certain that that's sufficient; let me know if you still think it isn't.

Reply:
I'll convert both games to use `get_all_par_prices`, rather than overriding `StockMarket`

Reply:
The conversion isn't tied to dfannius's PR, but certainly will make it easier!